integration_tests/tests/sqs.rs

```diff
@@ -141,3 +142,101 @@ fn sqs_timeout_test() {
     sqs.delete_queue(&queue_deletion_req).sync()
         .expect("delete queue failed");
 }
+
+#[test]
+fn sqs_bulk_roundtrip_tests() {
+	let _ = env_logger::try_init();
+	let sqs = SqsClient::simple(Region::UsEast1);
+
+	// create a new queue
+	let q_name = &format!("test_q_{}", time::get_time().sec);
```

Comment:
Can we use `rand::random::<u64>()` here instead of `time::get_time().sec`?

Using the current time results in tests that will run fine in sequence, but when run in parallel will mix up their queues. I had this problem with `sqs_roundtrip_tests` and `sqs_timeout_test`, which is why I made the changes there earlier.